thrift0.13/src/main/java/com/linecorp/armeria/internal/common/thrift/ThriftSerializationFormatProvider.java

```diff
@@ -63,4 +64,9 @@ private static MediaType create(String subtype) {
     private static MediaType create(String subtype, String protocol) {
         return create(subtype).withParameter("protocol", protocol);
     }
+
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper(this).toString();
```

Comment:
Does toStringHelper do anything without adding fields to it?

Reply:
Not exactly. 😄 Just removing the reference from the result string.
But yeah, this creates an additional instance so let me just use the string. 😄

Reply:
Changed my mind and just removed the method completely. 😉